### PR TITLE
[LE] Add timing to additional endpoints

### PR DESCRIPTION
**Pull request by Jolyon-S**
**Goals (and why)**:
There are two other responses that already have a leader ID as part of the response. I figured that we may as well get metrics on these endpoints too.

**Implementation Description (bullets)**:
* Extracted the leader ID from `leaderTime` and `refreshLocks`.

**Testing (What was existing testing like?  What have you done to improve it?)**:
* Added tests to confirm that all methods that we intend are being timed.

**Concerns (what feedback would you like?)**:
* This should not affect performance - right?

**Where should we start reviewing?**:
* `LeaderElectionReportingTimelockService`

**Priority (whenever / two weeks / yesterday)**:
Today

### CODE REVIEW

File: lock-api/src/test/java/com/palantir/lock/client/LeaderElectionReportingTimelockServiceTest.java [hunk 1]
```diff
@@ -49,13 +55,17 @@
 import org.mockito.stubbing.OngoingStubbing;
 
 public class LeaderElectionReportingTimelockServiceTest {
-    private static final UUID LEADER_1 = UUID.randomUUID();
+    private static final LeaderTime LEADER_1_TIME = LeaderTime.of(LeadershipId.random(), NanoTime.createForTests(999L));
```
Comment (Jolyon-S) [inline lock-api/src/test/java/com/palantir/lock/client/LeaderElectionReportingTimelockServiceTest.java:58]: the method to create from a UUID is not public.

File: lock-api/src/test/java/com/palantir/lock/client/LeaderElectionReportingTimelockServiceTest.java [hunk 2]
```diff
@@ -302,6 +335,11 @@ public void detectWhenOldLeaderBecomesLongTerm() {
         assertExpectedDuration(Instant.ofEpochMilli(3L), Instant.ofEpochMilli(15L));
     }
 
+    private void verifyEndpointRunsWithTiming(Runnable endpointCall) {
+        endpointCall.run();
+        verify(mockedClock, times(2)).instant();
```
Comment (jeremyk-91) [inline lock-api/src/test/java/com/palantir/lock/client/LeaderElectionReportingTimelockServiceTest.java:340]: a bit leaky, but I don't really have a better suggestion.

**Review by jeremyk-91 — APPROVED** [review]
🕐 0:03

👍

File: lock-api/src/test/java/com/palantir/lock/client/LeaderElectionReportingTimelockServiceTest.java (hunk 2 above)
Comment (Jolyon-S) [reply lock-api/src/test/java/com/palantir/lock/client/LeaderElectionReportingTimelockServiceTest.java:340]: I totally agree with both sentiments here :)

**Pull request merged**